tests/test_suite.py

```diff
@@ -55,11 +54,11 @@
                            metavar="LOGDIR",
                            default=None,
                            help="Directory to which all output is saved")
-common_parser.add_argument("--ansible",
-                           dest="ansible",
-                           action="store_true",
-                           default=False,
-                           help="Use ansible instead of bash remediations")
+common_parser.add_argument("--remediate-using",
+                           dest="remediate_using",
+                           default="oscap",
+                           choices=ssg_test_suite.oscap.REMEDIATION_RULE_RUNNERS.keys(),
+                           help="What type of remediations to use.")
```

Comment:
It would help to explain what each choice means, at least what is a difference between `oscap` and `bash`